identity_domain/src/commonMain/kotlin/net/thechance/mena/identity/domain/repository/UserRepository.kt

```diff
@@ -6,4 +6,9 @@ import net.thechance.mena.identity.domain.entity.User
 interface UserRepository {
     suspend fun getUser(): Flow<User?>
     suspend fun updateUser(user: User, shouldUpdateImage: Boolean, imageByteArray: ByteArray?)
+    suspend fun changePassword(
+        currentPassword: String,
+        newPassword: String,
+        confirmPassword: String
+    ): String
```

Comment:
I don't think we need to return a String, so please change it 

```suggestion
   suspend fun changePassword(
        currentPassword: String,
        newPassword: String,
        confirmPassword: String
    )
```